docs/language/functions/regexp_replace.md

```diff
@@ -58,3 +58,12 @@ echo '"option: value"' | zq -z 'yield regexp_replace(this,/(?P<key>\w+):\s+(?P<v
 ```mdtest-output
 "option=value"
 ```
+
+Wrap a named reference in curly braces to avoid ambiguity:
+```mdtest-command
```

Comment:
The other examples include an empty here.
```suggestion
Wrap a named reference in curly braces to avoid ambiguity:

```mdtest-command
```

Reply:
@nwt: Ah yes, right you are!